postgres/src/main/scala/com/lucidchart/relate/postgres/package.scala

```diff
@@ -0,0 +1,16 @@
+package com.lucidchart.relate
+
+import org.postgresql.util.PGInterval
+import scala.concurrent.duration.FiniteDuration
+import java.sql.PreparedStatement
+
+package object postgres {
```

Comment:
Never seen it shortened to postgres, but some searching show that it is done at least occasionally.